public/doc/swagger-2-v0.0.1.yaml

```diff
@@ -1655,6 +1655,8 @@ definitions:
       vendor:
         type: string
         example: Red Hat
+      schema:
+        type: string
```

Comment:
@bdunne since schema is a jsonb type would `type: object` be more appropriate here or is string fine?

Reply:
Yes, that's what I was thinking